src/main/java/salesforce/configuration/ConfigurationFile.java

```diff
@@ -0,0 +1,52 @@
+package salesforce.configuration;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Reads and gets the environment variables within properties file.
+ */
+public class ConfigurationFile {
+
+    private static final String CONFIG_FILE_PATH = "config.properties";
+    public static Properties CONFIG_PROPERTIES;
+
+    static {
+        try {
+            CONFIG_PROPERTIES = readFile();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
You should do something with the exception, not just printing it. You could throw it